stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/config/ServiceConfiguration.java

```diff
@@ -29,29 +29,16 @@
     private String rpcEndpoint;
     private BType requestType;
     private BType responseType;
-    private boolean clientStreaming;
-    private boolean serverStreaming;
 
-    public ServiceConfiguration(String rpcEndpoint, BType requestType, BType responseType, boolean clientStreaming,
-                                boolean serverStreaming) {
+    public ServiceConfiguration(String rpcEndpoint, BType requestType, BType responseType) {
         this.rpcEndpoint = rpcEndpoint;
         this.requestType = requestType;
         this.responseType = responseType;
```

Comment:
```suggestion
```

Reply:
I removed these two fields and the getters